package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "scripts": {
+    "clean": "rm -rf node_modules yarn-error.log css static/css *.log *.sqlite _site/ build/ .jekyll-metadata .bundle .repo en .extra",
+    "watch": "watch -p 'docs/**/*.md' -c 'yarn run build-css'",
+    "build-repo": "git config --global user.email 'you@example.com'; git config --global user.name 'Your Name'; git config --global color.ui false; ./repo init -u $(pwd); ./repo sync",
```

Comment:
It appears that `./repo init -u $(pwd)` will actually end up in it building from the `default.xml` in the **master branch** of the local repository. To fix that, you should also add an extra option: `--manifest-branch $(git rev-parse HEAD)` - so that it uses the local commit.

Also, could you use `--manifest-url` instead of `-u`? I think it aids understanding to use full option names in actual committed code or documentation.